.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+## Description
```

Comment:
This bring nothing useful IMHO, Github already display this for first time contributions. Could you remove it? thanks